src/main/java/com/minecolonies/coremod/colony/managers/ColonyPackageManager.java

```diff
@@ -52,6 +58,11 @@
      */
     private final Colony colony;
 
+    /**
+     * Random object.
+     */
+    private final Random random = new Random();
```

Comment:
we should use world random not our own thing

Reply:
kk